app/controllers/photos_controller.rb

```diff
@@ -22,6 +22,7 @@ def edit
   # POST /photos or /photos.json
   def create
     @photo = Photo.new(photo_params)
+    @photo.owner = current_user
```

Comment:
If the idea is to always set `owner` to `current_user`, should `photo_params` no longer permit `owner_id`?

Reply:
Great point. Will update.